### PR TITLE
[CPDLP-2007] Update API docs for request body for ecf completed declaration

### PR DESCRIPTION
**Pull request by leandroalemao**
### Context

- Ticket: [CPDLP-2007](https://dfedigital.atlassian.net/browse/CPDLP-2007)

Update API docs for request body for ecf completed declaration

### Changes proposed in this pull request

Make changes in the docs that shows `evidence_held` is a required field on ECF completed declaration request body.
Update release notes as well.

### Guidance to review

image

[CPDLP-2007]: https://dfedigital.atlassian.net/browse/CPDLP-2007?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by cwrw — COMMENTED** [review]
looks great, just one thing I noticed in v1/v2 (not introduced in this PR) is that for ECF we don't have `ECFParticipantDeclarationCompletedAttributesRequest` links as we do for NPQ (see https://ecf-review-pr-3031.london.cloudapps.digital/api-reference/reference-v2.html#schema-npqparticipantdeclarationattributes), could we add that for ECF please?

**Comment by leandroalemao** [comment]
> looks great, just one thing I noticed in v1/v2 (not introduced in this PR) is that for ECF we don't have `ECFParticipantDeclarationCompletedAttributesRequest` links as we do for NPQ (see https://ecf-review-pr-3031.london.cloudapps.digital/api-reference/reference-v2.html#schema-npqparticipantdeclarationattributes), could we add that for ECF please?

All done @cwrw tks for your review. 👍 😄

File: docs/Gemfile.lock [hunk 1]
```diff
@@ -6,12 +6,12 @@ GEM
       i18n (>= 1.6, < 2)
       minitest (>= 5.1)
       tzinfo (~> 2.0)
-    addressable (2.8.0)
-      public_suffix (>= 2.0.2, < 5.0)
-    autoprefixer-rails (10.4.2.0)
+    addressable (2.8.1)
```
Comment (cwrw) [inline docs/Gemfile.lock:9]: as you mentioned yesterday this was the issue, I've updated this and merged to main so hopefully if you rebase this is no longer needed

**Review by cwrw — APPROVED** [review]
Thanks @leandroalemao 🙌 just the one comment, and as we know updating the date till today

File: docs/Gemfile.lock (hunk 1 above)
Comment (leandroalemao) [reply docs/Gemfile.lock:9]: Should we still run `bundle update` tho?

Comment (cwrw) [reply docs/Gemfile.lock:9]: It worked for me with just making sure the ruby version was correct

**Pull request merged**